### ISSUE TITLE
HELP NEEDED: Translate Howdy to your language

### ISSUE THREAD

**Issue by boltgolt** [boltgolt/howdy#486]
The next release of Howdy will have support for internationalization and translations to any language are welcome! If you want to help [join us on Crowdin](https://crwd.in/howdy). If your language is not listed yet leave a comment and i'll add it ASAP

Thanks in advance!

**Comment by avelakafo** [comment]
I don't see Esperanto listed but I would be willing to undertake translating Howdy into it.

**Comment by boltgolt** [comment]
I've added esperanto! @avelakafo

**Comment by srueegger** [comment]
I started the German translation today and will finish it over the next week.
Good would be a German (Switzerland) translation option (which I would also directly translate).

**Comment by boltgolt** [comment]
Thanks a lot @srueegger! I've added German (Swiss) to the list of languages

**Comment by srueegger** [comment]
German Translation is complete.

Is it possible to copy the German translation 1: 1 to German (Swiss)? I just have to adjust a few strings.

**Comment by boltgolt** [comment]
Thanks @srueegger! I've copied the german translations to swiss, it did not catch any duplicated translations but you should be able to easily copy those from the translation memory

**Comment by srueegger** [comment]
Thanks for copying the language. German (Swiss) is also completely translated.

**Comment by Toufupi** [comment]
Translation of Chinese simplified is finished! Thanks for your great work  creating such a useful tool! By the way when will you release the new verision of howdy with gui? I think it is fancy too!

**Comment by boltgolt** [comment]
Thanks for your work @Toufupi! The GUI is going to be part of the larger v3.0 update which will break compatibility with older versions. Because of that i want to have as many breaking changes as possible in that one release and a couple of things are still being worked on. In short, no release date yet

**Comment by agnibha92** [comment]
How to request other languages?

**Comment by boltgolt** [comment]
@agnibha92 what language would you like me to add? There's unfortunately no way to let you add one yourself so just let me know what language you want

**Comment by agnibha92** [comment]
I can translate to Hindi/Bengali, If you want to add these to the list... @boltgolt

**Comment by boltgolt** [comment]
@agnibha92 I've added both languages to the list!

**Comment by agnibha92** [comment]
Thanks @boltgolt

**Comment by NedimBadzak** [comment]
I can translate to Bosnian/Croatian. @boltgolt

**Comment by samuelgregorovic** [comment]
@boltgolt i can translate to Slovak language if you add it there

**Comment by boltgolt** [comment]
@samuelgregorovic I've added Slovak!

**Comment by samuelgregorovic** [comment]
> @samuelgregorovic I've added Slovak!
thanks! i will start ASAP and let you know when im done

**Comment by boltgolt** [comment]
@samuelgregorovic no rush!

**Comment by samuelgregorovic** [comment]
@boltgolt translations are 100% done :)

**Comment by boltgolt** [comment]
@samuelgregorovic very quick, thank you!

**Comment by samuelgregorovic** [comment]
np ✌️

**Comment by liimee** [comment]
I'd love to translate it to Indonesian :)

**Comment by boltgolt** [comment]
@liimee added!

**Comment by simonpla** [comment]
Added a translation to Polish.

**Comment by boltgolt** [comment]
@simonpla thank you!

**Comment by simonpla** [comment]
@boltgolt No problem. :smile:

**Comment by crispy245** [comment]
@boltgolt Finished the Spanish translation! Hope to support the actual codebase soon; the project seems really interesting!

**Comment by boltgolt** [comment]
@crispy245 thanks a lot!

**Comment by chamik** [comment]
Translated to Czech. Glad I could help!

**Comment by boltgolt** [comment]
@chamik thank you!

**Comment by Mertszr02** [comment]
@boltgolt I finished the Turkish translation but Crowdin shows 0 approved strings. I'm guessing approval is needed before the strings can be implemented. Is there anything I can do to help with that? 
btw thank you for creating Howdy it is incredible. I'm in the process of learning how Howdy works so I can help speed up the 3.0 release by Committing code.

**Comment by boltgolt** [comment]
@Mertszr02 that just means that Crowdin is waiting on other Turkish speakers to verify your translation. I'm sure your translation is good so I'm going to import it regardless.

Good to hear! We especially need people to test the changes in #484

**Comment by Itotutona** [comment]
I want to translate howdy in french

**Comment by boltgolt** [comment]
@Itotutona I'd be very grateful if you could [verify the translations](https://translate.howdy.boltgolt.nl/project/howdy/fr) already done

**Comment by boltgolt** [comment]
Thanks @gogoname! I'll take a look what i can do for LTR languages

**Comment by NP-chaonay** [comment]
I want to request "Thai" language (language code: th)

I don't sure if I can able to do all 100% but I think most translation is not much difficult.

Thanks.

**Comment by boltgolt** [comment]
Added @NP-chaonay, thanks!

**Comment by GWFrank** [comment]
The rest of Traditional Chinese translation is done.
Would be great if someone can help verifying and do some polishing.

Glad that I can participate in this amazing project!

**Comment by boltgolt** [comment]
@GWFrank thank you!

**Comment by antival3nt** [comment]
Hey @boltgolt 
No software really used to be translated to Danish, I'd love to do it here, if it's okay? :)

**Comment by saltedcoffii** [comment]
Hey @boltgolt could you divide Spanish into "Spanish, Spain", and "Spanish, Latin America"? You could also further subdivide latin american spanish into "Spanish, Mexico" and "Spanish, Argentina" if you wanted.

**Comment by gimdh** [comment]
I have translated Korean, but there are couple of ambiguities that require confirmation. I intentionally left CLI script as English as I believe most Koreans willing to use CLI is comfortable with English. @boltgolt

**Comment by boltgolt** [comment]
Sorry for the delayed response!

@antival3nt Of course! Danish should be ready to translate on Crowdin

@gimdh Thanks! Are the 55% of the strings that are not translated just the CLI?

**Comment by gimdh** [comment]
@boltgolt Yes. _Main UI / Onboarding UI / UI scrpts_ are completely translated, though few clarifications would make it more clear.

**Comment by drcor** [comment]
Hi @boltgolt 
I finished the missing translations for Portuguese (Portugal), now it is 100% done :)

**Comment by Edyburr** [comment]
Hey @boltgolt !
I started the Romanian translation today and should be able to finish it by the end of the week.
Cheers!

**Comment by BloodDrag0n** [comment]
Is Tamil translation done or you guys need any help?

**Comment by kuracode** [comment]
Hi @boltgolt 
Started on the Finnish translation. Might be able to get it done this week.

**Comment by boltgolt** [comment]
Thanks so much for the help @drcor @Edyburr @kuracode!
@BloodDrag0n tamil is currently at 0%, so i'd be very happy if you could help there

**Comment by blackinitial** [comment]
I will continue the Indonesian translation

**Comment by zhujinle** [comment]
Voted and polished for Simplified Chinese and Traditional Chinese translation.
Thanks for your nice tool and waiting for the release of new Howdy~
Is there any news about the release?

**Comment by andywarh0l** [comment]
I'd like to translate it into Italian if someone hasn't already done so @boltgolt

**Comment by joezhouchenye** [comment]
So, this is for the beta version with GUI support? For the master branch, I just need to modify the pam.py to translate into my language.

**Comment by mvanzulli** [comment]
I can add Euskera if anyone already done it @boltgolt

**Comment by Jhonata-Tirloni** [comment]
Hi @boltgolt , the link seens broken. I can help with brazilian portuguese and italian.

**Comment by Bebegick** [comment]
Hello boltgolt,
I'm interested in contributing to this project, specifically by helping to fix/translate it into Indonesian. Could you please let me know if this issue is still open?

Thank you, and I look forward to your response!

**Comment by IlNonoP** [comment]
I can't access to the link... can i contribute to italian?

**Comment by TahierJerr** [comment]
I would like to help with Dutch!

**Comment by binarydepth** [comment]
Do you have Spanish done?

**Comment by Notu3** [comment]
Hello Boltgolt i wanna help with the Spanish (Europe) Translation if it isnt already done.  ¿How can i help? the crowdin link isnt working

**Comment by gg582** [comment]
Hello, boltgolt. I'd like to assist you with Korean(South Korea) translation if it is not already done. May I ask for the link?

**Comment by Agr96** [comment]
hi @boltgolt 

im looking your current pull and interested for contributing of translation to **indonesian** so let me know if you still looking forward for it.

dear, 
andrew
